pybaseball/playerid_lookup.py

```diff
@@ -18,43 +19,51 @@ def chadwick_register(save: bool = False) -> pd.DataFrame:
 
     print('Gathering player lookup table. This may take a moment.')
     s = requests.get(url).content
-    mlb_only_cols = ['key_retro', 'key_bbref', 'key_fangraphs', 'mlb_played_first', 'mlb_played_last']
+    mlb_only_cols = ['key_retro', 'key_bbref',
```

Comment:
I don't like this. This was col 103 and splitting this doesn't really harm readability

Reply:
Yeah I'm not sure why it's splitting early there.